src/cljs/nr/utils.cljs

```diff
@@ -23,6 +24,11 @@
 (def rotated-span
   [:span.casual {:title "Rotated"} " " rotated-dot])
 
+(defn deck-points-card-span [points]
+  [:span.legal {:title (when points
```

Comment:
Should you make the span when there are no points to display?

Reply:
I think it is useful since it allows you to display the symbol without a value when you hover over it. I use this for example in the header `Deck points: 4/7 ❖`. Since that symbol is just visual on that line I don't give it a value so nothing displays when you hover over it. Could be worth just displaying the point limit even though it is listed next to it. I feel like there was one other place I do this, but I can't remember right now.